micropython/senml/senml/senml_kpn_names.py

```diff
@@ -0,0 +1,46 @@
+# _  __  ____    _   _
+# | |/ / |  _ \  | \ | |
+# | ' /  | |_) | |  \| |
+# | . \  |  __/  | |\  |
+# |_|\_\ |_|     |_| \_|
+#
+# (c) 2018 KPN
+# License: MIT license.
+# Author: Jan Bogaerts
+#
+# sensor names
+
+from senml.senml_unit import enum
+
+
+SenmlNames = enum(
+    KPN_SENML_PRESSURE="pressure",
```

Comment:
As mentioned elsewhere, IMO this file can be removed.  Users can define their own names simply as strings.